src/pudl/package_data/meta/xlsx_maps/eia860/column_maps/generator_proposed.csv

```diff
@@ -1,63 +1,63 @@
+energy_source_code_2,ensource2,ensource2,ensource2,proposed_energy_source_2,proposed_energy_source_2,proposed_energy_source_2,energy_source_2,energy_source_2,energy_source_2,energy_source_2,energy_source_2,energy_source_2,energy_source_2,energy_source_2,energy_source_2,energy_source_2,energy_source_2,energy_source_2,energy_source_2
+energy_source_code_3,ensource3,ensource3,ensource3,proposed_energy_source_3,proposed_energy_source_3,proposed_energy_source_3,energy_source_3,energy_source_3,energy_source_3,energy_source_3,energy_source_3,energy_source_3,energy_source_3,energy_source_3,energy_source_3,energy_source_3,energy_source_3,energy_source_3,energy_source_3
+energy_source_code_4,ensource4,ensource4,ensource4,proposed_energy_source_4,proposed_energy_source_4,proposed_energy_source_4,energy_source_4,energy_source_4,energy_source_4,energy_source_4,energy_source_4,energy_source_4,energy_source_4,energy_source_4,energy_source_4,energy_source_4,energy_source_4,energy_source_4,energy_source_4
+energy_source_code_5,ensource5,ensource5,ensource5,proposed_energy_source_5,proposed_energy_source_5,proposed_energy_source_5,energy_source_5,energy_source_5,energy_source_5,energy_source_5,energy_source_5,energy_source_5,energy_source_5,energy_source_5,energy_source_5,energy_source_5,energy_source_5,energy_source_5,energy_source_5
+energy_source_code_6,ensource6,ensource6,ensource6,proposed_energy_source_6,proposed_energy_source_6,proposed_energy_source_6,energy_source_6,energy_source_6,energy_source_6,energy_source_6,energy_source_6,energy_source_6,energy_source_6,energy_source_6,energy_source_6,energy_source_6,energy_source_6,energy_source_6,energy_source_6
+energy_source_1_transport_1,transmode1,transmode1,transmode1,proposed_transportation_1a,proposed_transportation_1a,proposed_transportation_1a,,,,,,,,,,,,,
+energy_source_1_transport_2,,,,proposed_transportation_1b,proposed_transportation_1b,proposed_transportation_1b,,,,,,,,,,,,,
+energy_source_1_transport_3,,,,proposed_transportation_1c,proposed_transportation_1c,proposed_transportation_1c,,,,,,,,,,,,,
+energy_source_2_transport_1,transmode2,transmode2,transmode2,proposed_transportation_2a,proposed_transportation_2a,proposed_transportation_2a,,,,,,,,,,,,,
+energy_source_2_transport_2,,,,proposed_transportation_2b,proposed_transportation_2b,proposed_transportation_2b,,,,,,,,,,,,,
+energy_source_2_transport_3,,,,proposed_transportation_2c,proposed_transportation_2c,proposed_transportation_2c,,,,,,,,,,,,,
+multiple_fuels,,,,,,,,,multiple_fuels,multiple_fuels,multiple_fuels,multiple_fuels,,,,multiple_fuels,multiple_fuels,multiple_fuels,multiple_fuels
+ownership_code,owner,owner,owner,owner,owner,owner,ownership,ownership,ownership,ownership,ownership,ownership,ownership,ownership,ownership,ownership,ownership,ownership,ownership
+turbines_num,propwind,propwind,propwind,proposed_turbines,proposed_turbines,proposed_turbines,turbines,turbines,turbines,turbines,turbines,turbines,,,,,,,

```

Comment:
Are these early years referring to **wind** turbines? Is that true for the latter years as well?